### PR TITLE
Game: create first flow.

### PR DESCRIPTION
**Pull request by kbenarie**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (OmriKaplan) [inline .idea/.gitignore]: Let's remove this file and the rest of `.idea` folder, so we don't have development environment configurations tracked.

**Review by OmriKaplan — CHANGES_REQUESTED** [review]
Good job! 🇺🇬 
Let's change the PR to include the following commits:
1. adds only `.gitignore`.
2. all of the code changes to the new design including the new test.

File: .idea/.gitignore (hunk 1 above)
Comment (kbenarie) [reply .idea/.gitignore]: Done.

**Review by OmriKaplan — APPROVED** [review]
Good job! 💯

**Pull request merged**